config/management/src/key.rs

```diff
@@ -0,0 +1,73 @@
+// Copyright (c) The Libra Core Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::{error::Error, SecureBackends};
+use libra_crypto::ed25519::Ed25519PublicKey;
+use libra_secure_storage::{Storage, Value};
+use std::convert::TryInto;
+use structopt::StructOpt;
+
+#[derive(Debug, StructOpt)]
+pub struct AssociationKey {
```

Comment:
Nit: What's the motivation behind adding another layer of indirection? i.e., we now have Command::*Key (which provides an execute() call), and we also have crate::Key::*Key which provides execute(). I can see that it makes a little bit more sense in other cases (e.g., genesis, waypoint) where there's more going on, but here it might be a little bit overkill? At the same time though, I can see why you might do it for consistency... (no changes required, I just found it a little bit of an interesting pattern!)

Reply:
Well actually at this point in time, we could unify the code in lib if all the executes returned the same type. I'm not sure it is worth it. The second part was just consistency. I think it is better to follow this type of formula than to try to have 40+ line exceptions.